AnoopExercises/Sources/PhotosViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+// Copyright © 2022 Surya Software Systems Pvt. Ltd.
+//
+
+import UIKit
+
+class PhotosViewController: UIViewController {
+    // MARK: - views
+
+    private weak var collectionView: UICollectionView!
+
+    // MARK: - variables
+
+    var store: PhotoStore!
+    let photoDataSource = PhotoDataSource()
+
+    // MARK: - View Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        navigationItem.title = R.string.localizable.rootViewTitle()
+        navigationItem.backBarButtonItem = UIBarButtonItem(title: R.string.localizable.backButton(), style: .plain, target: nil, action: nil)
+
+        let constraints = setupCollectionView()
+        NSLayoutConstraint.activate(constraints)
+
+        // add list of photos
+        fetchPhotos()
+    }
+
+    // MARK: - Setup
+
+    private func setupCollectionView() -> [NSLayoutConstraint] {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+        layout.itemSize = CGSize(width: 90, height: 90)
+        layout.minimumLineSpacing = 2
+        layout.minimumInteritemSpacing = 2
+        layout.sectionInset = UIEdgeInsets(top: 2, left: 2, bottom: 2, right: 2)
+
+        let collecitonView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collecitonView.configureView { cv in
+            cv.register(PhotoCollectionViewCell.self, forCellWithReuseIdentifier: PhotoCollectionViewCell.cellIdentifier)
+        }
+
+        collectionView = collecitonView
+        collectionView.delegate = self
+        collectionView.dataSource = photoDataSource
```

Comment:
This should also go inside `configureView`

Reply:
sure ill make that change